erpnext/accounts/report/financial_statements.py

```diff
@@ -258,3 +258,35 @@ def get_columns(period_list):
 		})
 
 	return columns
+
+
+def get_data_account_type(company, account_type, period_list):
+    data = {}
+    for period in period_list:
+        gl_sum = frappe.db.sql_list("""select sum(credit) - sum(debit) FROM `tabGL Entry`
+                                    where company='%s' and posting_date >= '%s' and posting_date <= '%s' and
+                                    account in ( SELECT name FROM tabAccount WHERE account_type = '%s')""" %
+                                    (company, period['from_date'], period['to_date'], account_type))
```

Comment:
@anandpdoshi I tried to look other sql_list code in erpnext. This is at accounts/utils.py line 252.
```
def remove_against_link_from_jv(ref_type, ref_no):
	linked_jv = frappe.db.sql_list("""select parent from `tabJournal Entry Account`
		where reference_type=%s and reference_name=%s and docstatus < 2""", (ref_type, ref_no))
```

We have the same syntax in making an sql call. Can you give me some idea how to avoid possible injection?